cvat-ui/src/components/header/header.tsx

```diff
@@ -62,6 +67,34 @@ function HeaderContainer(props: Props): JSX.Element {
                 >
                     Tasks
                 </Button>
+                <Popover placement="bottom" title={content} content={text} trigger="click" overlayStyle={{width : "50vw"}}>
+                <Button
+                    className='cvat-header-button'
+                    type='link'
+                    value='tasks'
+                    onClick={
+                        (): void => {
+                        const t = localStorage.getItem('token');
+                        const token ='Token '+ t.replace(/['"]+/g, '')
+                        const data = axios.get(`${serverHost}/api/v1/server/about`,{ headers: {Authorization : token}}).then(
```

Comment:
@LiSa20120 

We shouldn't make server requests here. 
There are two problems:
1. To be consistent and do not do extra requests, we should use ``redux`` here (create reducer [for example ServerInfoReducer], add it to CVAT storage, add some actions for the reducer) and provide some props from the storage to the component. Look for example how we get users (usersReducer, usersActions) in such a scheme. 
2. You do not need to do request manually with axios and add an authorization token. Just use cvat-core to get this data:
```js
const core = getCore();
core.server.about()
```